source/elements/oneDNN/source/primitives/normalization.rst

```diff
@@ -0,0 +1,25 @@
+..
+  Copyright 2019-2020 Intel Corporation
+
+.. default-domain:: cpp
```

Comment:
In source/conf/common.py we set the primary_domain to cpp so this isn't necessary. It's harmless to commit but you may want to delete them in the future.

Reply:
Understood. I was just adding them in case we get a heterogeneous API and the `primiary_domain` goes away.

Reply:
OK. It is up to you.